tests/cxx/enums.cc

```diff
@@ -42,6 +42,13 @@ Struct1::IndirectEnum3 ie3;
 // IWYU: UnnamedEnumItem2 is...*enums-i4.h
 auto ie4 = UnnamedEnumItem2;
 
+// No need even in opaque declaration if enumeration or enumerator isn't named
+// explicitly (it already should be elsewhere in the scope). No matter, whether
+// enum has fixed type or not, used in sizeof operator, pointer arithmetics
```

Comment:
arithmetics -> arithmetic (it's uncountable in English)

Reply:
Thank you! In Russian too. Just a wrong word formation after "mathematics", "physics" etc.